### DISCUSSION TITLE
Great Project!

### DISCUSSION THREAD

**Discussion by bjb28** [rdeepak2002/reddit-place-script-2022#443] [General]
Thanks for a great project! Sad to see r/place is starting to wind down. Great few days of coding that I got to learn a lot from.

**Comment by Risewill23** [comment]
I agree I forgot a lot about python because I mostly use things like node.js and C# instead of python

**Comment by Skyline-9** [comment]
It was really fun learning from everyone. With so many contributors, there were so many cool ideas
